README.md

```diff
@@ -55,7 +55,8 @@ If you do not have a patched font installed, you can override or remove any icon
 2. Add the Catppuccin plugin:
 
 ```bash
-set -g @plugin 'catppuccin/tmux'
+set -g @plugin 'catppuccin/tmux#latest' # See https://github.com/catppuccin/tmux/tags for additional tags
+# set -g @plugin 'catppuccin/tmux' # main branch
```

Comment:
If people rename the plugin locally to `tmux-catppuccin`, do the tags still work for them?

Reply:
Github doesn't copy tags on fork so no but when forking you also pin the a specific commit so don't see a case for versioning then.